Include/internal/pycore_unionobject.h

```diff
@@ -12,6 +12,9 @@ PyAPI_FUNC(PyObject *) _Py_Union(PyObject *args);
 PyAPI_DATA(PyTypeObject) _Py_UnionType;
 PyAPI_FUNC(PyObject *) _Py_union_type_or(PyObject* self, PyObject* param);
 
+PyAPI_FUNC(PyObject *) _Py_subs_parameters(PyObject *, PyObject *, PyObject *, PyObject *);
+PyAPI_FUNC(PyObject *) _Py_make_parameters(PyObject *);
```

Comment:
Thanks to Pablo's comments above, I finally understand what the check is doing. Seems that release branches have a frozen generated list of exported symbols which are tested against. So there a few solutions:

1. Follow instructions at https://bugs.python.org/issue43725#msg390213 and regen the exported symbols. Though this looks somewhat time consuming.
2. Add ignorefile.
3. Remove the `PYAPI_FUNC` (I like this most):
```suggestion
PyObject *_Py_subs_parameters(PyObject *, PyObject *, PyObject *, PyObject *);
PyObject *_Py_make_parameters(PyObject *);
```
I don't have the proper environment to test that 3. will make the test pass. But I think it should.